js/foundation.tabs.js

```diff
@@ -65,7 +65,6 @@ class Tabs {
 
       $tabContent.attr({
         'role': 'tabpanel',
-        'aria-hidden': !isActive,
```

Comment:
or wait... I guess if default behavior is display:none that is enough to tell the browser these are hidden...